cloud-gateway-service/src/main/java/org/zowe/apiml/cloudgatewayservice/service/CertificateChainService.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.cloudgatewayservice.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.bouncycastle.openssl.jcajce.JcaPEMWriter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.zowe.apiml.security.HttpsConfig;
+import org.zowe.apiml.security.SecurityUtils;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.security.cert.Certificate;
+
+/**
+ * This service provides gateway's certificate chain which is used for the southbound communication
+ */
+@Service
+@Slf4j
+public class CertificateChainService {
+
+    @Value("${server.ssl.keyStore:#{null}}")
+    private String keyStore;
```

Comment:
this would need to be updated when separate configuration of keystore for client is possible, it is going to be added with Central Apiml Registry

Reply:
Will add TODO to address this once the PR with Central Apiml Registry is merged.